OneginiExampleAppSwift/Presenters/LoginPresenter.swift

```diff
@@ -67,12 +67,11 @@ class LoginPresenter: LoginPresenterProtocol {
         profiles = loginInteractor.userProfiles()
```

Comment:
userProfiles() -> userProfiles in setupLogin/reloadProfile.

haven't you even checked whether it compiles? 🤦🏼

Reply:
That happens when you switching projects over and over again .